website/src/categories/components/tooltip/tooltip.md

```diff
@@ -3,13 +3,15 @@ title: Tooltip
mponent representing tooltip component card.</desc><path fill="#1138AD" d="M160.105 80.579c-.735-.001-1.435-.272-1.919-.743l-6.262-6.093c-1.018-.99-2.83-.99-3.848 0l-6.262 6.093c-.485.47-1.184.742-1.919.743h-25.263a2.526 2.526 0 0 0-2.527 2.526v35.369a2.526 2.526 0 0 0 2.527 2.526h70.736a2.526 2.526 0 0 0 2.527-2.526V83.105a2.526 2.526 0 0 0-2.527-2.526h-25.263Z"/><path fill="#F4F4F4" d="M130.689 105.842V96.96h-3.001v-1.061h7.275v1.06h-3.001v8.883h-1.273ZM138.491 106.024a3.308 3.308 0 0 1-1.683-.455c-.515-.303-.934-.742-1.258-1.318-.313-.576-.47-1.269-.47-2.077 0-.829.157-1.531.47-2.107.324-.576.743-1.015 1.258-1.319a3.317 3.317 0 0 1 1.683-.454c.606 0 1.167.151 1.683.454.515.303.929.743 1.242 1.319.324.576.486 1.278.486 2.107 0 .808-.162 1.501-.486 2.077-.313.576-.727 1.015-1.242 1.318a3.256 3.256 0 0 1-1.683.455Zm0-1.031c.637 0 1.147-.257 1.531-.773.394-.525.591-1.207.591-2.046 0-.849-.197-1.536-.591-2.061-.384-.526-.894-.789-1.531-.789-.627 0-1.137.263-1.531.789-.394.525-.591 1.212-.591 2.061 0 .839.197 1.521.591 2.046.394.516.904.773 1.531.773ZM146.706 106.024a3.302 3.302 0 0 1-1.682-.455c-.515-.303-.935-.742-1.258-1.318-.313-.576-.47-1.269-.47-2.077 0-.829.157-1.531.47-2.107.323-.576.743-1.015 1.258-1.319a3.31 3.31 0 0 1 1.682-.454 3.26 3.26 0 0 1 1.683.454c.515.303.93.743 1.243 1.319.323.576.485 1.278.485 2.107 0 .808-.162 1.501-.485 2.077-.313.576-.728 1.015-1.243 1.318a3.25 3.25 0 0 1-1.683.455Zm0-1.031c.637 0 1.147-.257 1.531-.773.395-.525.592-1.207.592-2.046 0-.849-.197-1.536-.592-2.061-.384-.526-.894-.789-1.531-.789-.626 0-1.136.263-1.53.789-.395.525-.592 1.212-.592 2.061 0 .839.197 1.521.592 2.046.394.516.904.773 1.53.773ZM153.376 106.024c-.465 0-.804-.141-1.016-.424-.202-.293-.303-.708-.303-1.243V95.05h1.243v9.398c0 .192.035.333.106.424a.316.316 0 0 0 .243.121h.106c.04-.01.096-.02.167-.03l.166.94c-.081.04-.177.071-.288.091a2.46 2.46 0 0 1-.424.03ZM158.24 106.024c-.788 0-1.339-.227-1.652-.682-.304-.455-.455-1.046-.455-1.773v-4.078h-1.092v-.94l1.152-.076.152-2.061h1.046v2.061h1.986v1.016h-1.986v4.093c0 .454.081.808.242 1.061.172.242.47.364.895.364.131 0 .273-.021.424-.061l.409-.137.243.94a5.13 5.13 0 0 1-.667.182 2.734 2.734 0 0 1-.697.091ZM161.042 105.842v-7.367h1.243v7.367h-1.243Zm.637-8.882a.885.885 0 0 1-.606-.212c-.152-.152-.228-.354-.228-.607 0-.242.076-.44.228-.591a.85.85 0 0 1 .606-.227.8.8 0 0 1 .591.227.773.773 0 0 1 .243.591c0 .253-.081.455-.243.606a.833.833 0 0 1-.591.213ZM164.773 108.95V98.475h1.03l.106.85h.046a5.69 5.69 0 0 1 1.091-.728 2.78 2.78 0 0 1 1.258-.303c.95 0 1.673.343 2.168 1.03.495.677.743 1.587.743 2.729 0 .828-.152 1.541-.455 2.137-.293.596-.682 1.051-1.167 1.364-.475.313-.996.47-1.562.47-.343 0-.687-.076-1.03-.227a5.296 5.296 0 0 1-1.016-.622l.03 1.289v2.486h-1.242Zm3.046-3.972c.607 0 1.107-.257 1.501-.773.404-.525.606-1.243.606-2.152 0-.809-.151-1.46-.455-1.956-.293-.505-.788-.757-1.485-.757-.313 0-.632.085-.955.257-.313.172-.652.42-1.016.743v3.865c.334.283.657.485.971.606.313.112.591.167.833.167Z"/></svg>
```

Comment:
It looks like for tabs there is wrong SVG added (accordion)?

<img width="447" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/4be470ed-717c-41a0-b8ef-dba51c61b4f2">